app/models/event.rb

```diff
@@ -405,4 +406,12 @@ def set_signature
     self.signature = true
   end
 
+  def globally_visible_nil?
+    globally_visible.nil?
+  end
+
+  def set_globally_visible_default
+    self.globally_visible = Settings.event.globally_visible_by_default
```

Comment:
Hier bin ich auch nicht so sicher ob ich das so beibehalten würde. Wir werden so oder so nie 100% Abdeckung aller Events erreichen mit dieser Methode, es wird immer ein paar alte Events geben die das nicht gesetzt haben. Wenn wir `nil`-Werte abschaffen wollen wäre es effektiver, das einfach in der DB-Migration zu machen. Wenn wir `nil`-Werte und globales Fallback-Setting unterstützen wollen, braucht es das hier auch nicht.

Reply:
gut, dass pass ich dann noch an